src/ape_ethereum/ecosystem.py

```diff
@@ -97,6 +97,7 @@ class NetworkConfig(PluginConfig):
     block_time: int = 0
     transaction_acceptance_timeout: int = DEFAULT_TRANSACTION_ACCEPTANCE_TIMEOUT
     default_transaction_type: TransactionType = TransactionType.DYNAMIC
+    max_receipt_retries = 20
```

Comment:
I am a little confused about the name because it is being used when we are fetching the transaction; not when we are fetching the receipt.

Are we sure we are using this config in the right spot and/or is this the correct name describing when in the tx lifecycle that it gets used?

Reply:
As for the config being in the right spot, that was Doggie's suggestion, if we made this a provider config we would have to update all the provider plugins

As for naming, I think you're right that I picked the wrong name here, I'll rename this to `max_get_transaction_retries`